libcudacxx/.upstream-tests/test/std/utilities/time/date.time/ctime.pass.cpp

```diff
@@ -51,7 +51,9 @@ int main(int, char**)
     unused(tmspec); // Prevent unused warning
 #endif
 
+#ifndef TEST_COMPILER_CLANG_CUDA
     static_assert((cuda::std::is_same<decltype(cuda::std::clock()), cuda::std::clock_t>::value), "");
```

Comment:
This one might be problemmatic, we get different `clock_t` types (int vs long)

I did not investigate thoroughly why